detection_rules/eswrap.py

```diff
@@ -233,13 +233,14 @@ def normalize_file(events_file):
 @click.option('--cloud-id', callback=set_param_values, expose_value=True)
 @click.option('--user', '-u', callback=set_param_values, expose_value=True, hide_input=False)
 @click.option('--password', '-p', callback=set_param_values, expose_value=True, hide_input=True)
-def kibana_upload(toml_files, kibana_url, cloud_id, user, password):
+@click.option('--space', callback=set_param_values, expose_value=True, hide_input=False)
+def kibana_upload(toml_files, kibana_url, cloud_id, user, password, space):
     """Upload a list of rule .toml files to Kibana."""
     from uuid import uuid4
     from .packaging import manage_versions
     from .schemas import downgrade
 
-    with Kibana(cloud_id=cloud_id, url=kibana_url) as kibana:
+    with Kibana(cloud_id=cloud_id, url=kibana_url, kibana_space=space) as kibana:
```

Comment:
s/kibana_space/space?

Reply:
I don't follow? `kibana_space` is the name of the variable where as `space` is the value for the variable.

Resulting output is correct (in my testing)

Reply:
oh, i was more just saying, since `kibana_space` is the name of a parameter for `Kibana.__init__`, it was slightly redundant, so  it could just be `space` in both.